### PR TITLE
added mod version display to mod option menu

### PR DESCRIPTION
**Pull request by EL20202**
Does as title suggests - simply makes the mod's version appear in the option menu.

Preview of change:

image

### CODE REVIEW

File: assets/mods/simplify/mod.js [hunk 1]
```diff
@@ -804,7 +807,18 @@
...
-			
+
+					if(this.option.version != null) {
+						this.versionText = new sc.TextGui(this.option.version, {
+							font: sc.fontsystem.tinyFont,
+							speed: ig.TextBlock.SPEED.IMMEDIATE
+						});
+						this.versionText.setAlign(ig.GUI_ALIGN.X_RIGHT, ig.GUI_ALIGN.Y_BOTTOM);
+						console.log(this.hook.size.x, this.hook.size.x / 2, this.hook.size.x - 48, this.hook.size.y)
+						this.versionText.setPos(38, 5);
```
Comment (dmitmel) [inline assets/mods/simplify/mod.js:818]: Can you please calculate this position from other coordinates? Like this:

```js
let lineHook = this.hook.children[1]; // this is already defined a little above
let slopeHook = this.hook.children[2];
// ...
// this code:
this.versionText.setPos(slopeHook.pos.x - this.versionText.hook.size.x - 4, slopeHook.pos.y + 2);
```

File: assets/mods/simplify/mod.js [hunk 2]
```diff
@@ -804,7 +807,18 @@
 						this.addChildGui(this.iconGui);
 						this.nameGui.hook.pos.x += this.iconGui.hook.pos.x + this.iconGui.hook.size.x;
 					}
-			
+
+					if(this.option.version != null) {
+						this.versionText = new sc.TextGui(this.option.version, {
+							font: sc.fontsystem.tinyFont,
+							speed: ig.TextBlock.SPEED.IMMEDIATE
```
Comment (dmitmel) [inline assets/mods/simplify/mod.js:814]: Also, I thought that this (speed) was the default

**Review by dmitmel — CHANGES_REQUESTED** [review]

File: assets/mods/simplify/mod.js (hunk 1 above)
Comment (dmitmel) [reply assets/mods/simplify/mod.js:818]: Oh, and please remove the `console.log`

Comment (EL20202) [reply assets/mods/simplify/mod.js:818]: 🤦 I thought I removed that... And yeah I can see about that.

**Comment by EL20202** [comment]
There, that should be good then, right?

**Pull request merged**